src/lib/github/createdIssuesList.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * extracts a list of open issues created by the app
+ */
+
+export interface openIssues {
```

Comment:
rename this to `OpenIssue` . `OpenIssues` is more appropriate for an array of objects

Reply:
please address this to @lubegasimon